src/current/cockroachcloud/managing-access.md

```diff
@@ -32,26 +32,26 @@ An [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administra
 
 1. If you are a member of multiple organizations, navigate to the organization to which you want to invite a team member. You can navigate to the correct organization by using the drop-down box in the top-right corner.
 1. On the **Access Management** page, under the *Members* tab, click **Invite**.
-1. In the **Email Address** field, enter the email address of the team member you want to invite. By default, a user is assigned the [Organization member]({% link cockroachcloud/authorization.md %}#organization-member) role; this default role grants no access. After the user accepts the invitation, an [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator) can grant them additional roles.
+1. In the **Email Address** field, enter the email address of the team member you want to invite. By default, a user is assigned the [Organization member]({% link cockroachcloud/authorization.md %}#organization-member) role; this default role grants no access. After the user is invited, an [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator) can [grant additional roles](#change-a-team-members-roles).
```

Comment:
Other admin roles are also able to grant additional roles. I'm not sure the nuance matters to users at this point, but for example, a Cluster Admin can give the user permissions for any clusters they have admin permissions on.

Reply:
Maybe I should change it to "you can grant..." because if you have permission to invite a user, you have permission to give them more roles. WDYT?

Reply:
That makes sense to me!